vlib/v/checker/tests/option_ptr_without_unwrapp_err.out

```diff
@@ -1,6 +1,6 @@
 vlib/v/checker/tests/option_ptr_without_unwrapp_err.vv:7:13: error: cannot use `?&Node` as `Node`, it must be unwrapped first in argument 1 to `set_trace`
     5 | fn set_trace(n Node) {
-    6 |     if n.parent != none {
+    6 |     if n.parent == none {
     7 |         set_trace(n.parent)
       |                   ~~~~~~~~
```

Comment:
That is a now a wrong error- if the condition is `if n.parent == none {`, then `n.parent` is sure to be `none`, and it is already unwrapped.

Reply:
Imho restore that .vv/.out pair as it was, do not invert the condition here - it worked well before.

Reply:
> Imho restore that .vv/.out pair as it was, do not invert the condition here - it worked well before.

But there is no error restoring it.